### PR TITLE
Add TypeScript definition

### PR DESCRIPTION
**Pull request by BendingBender**
Was following @sindresorhus's [styleguide](https://github.com/sindresorhus/typescript-definition-style-guide).

### CODE REVIEW

**Review by sindresorhus — APPROVED** [review]

File: index.test-d.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import * as minimist from 'minimist';
```
Comment (vadimdemedes) [inline index.test-d.ts:1]: I'm not familiar with TS much, what's this file for?

**Comment by vadimdemedes** [comment]
Great PR, thank you @BendingBender!

File: index.test-d.ts (hunk 1 above)
Comment (sindresorhus) [reply index.test-d.ts:1]: You test code with AVA. This tests the TS types, to ensure they're correct.

See https://github.com/SamVerschueren/tsd-check

Comment (vadimdemedes) [reply index.test-d.ts:1]: 👍

**Comment by vadimdemedes** [comment]
Released as 4.0.1!

**Comment by BendingBender** [comment]
@vadimdemedes If you need help supporting the typedefs, just ping me.

**Pull request merged**